File: Com.Fubei.Api.OpenApi/Biz/AgentApi/AgentApiAttribute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Com.Illuminati.Galileo;
using Com.Illuminati.Galileo.Attr;

namespace Com.Fubei.Api.OpenApi.Biz
{
    public class AgentApiAttribute : PostApiAttribute
    {
        public AgentApiAttribute()
        {
            Path = "/gateway/agent";
            Category = GalileoApiConfig.Category.AgentApi;
        }
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/AgentApi/AgentApiAttribute.vb
Imports Com.Illuminati.Galileo
Imports Com.Illuminati.Galileo.Attr

Namespace Biz.V2

    Public Class AgentApiAttribute
        Inherits PostApiAttribute

        Public Sub New()
            Path = "/gateway/agent"
            Category = GalileoApiConfig.Category.AgentApi
        End Sub
    End Class
End Namespace
